crates/astria-conductor/Dockerfile

```diff
@@ -34,8 +34,11 @@ RUN cargo chef prepare --recipe-path recipe.json
 FROM chef as builder
 COPY --from=planner /build/recipe.json recipe.json
 
+RUN apt-get update \
+    && apt-get install -y \
+    librocksdb-dev
+    
 ARG TARGETPLATFORM
-ENV ROCKSDB_LIB_DIR=/usr/lib
```

Comment:
I believe this was added to improve build times? @SuperFluffy knows context

Reply:
yeah i re-added this and now the image fails to build lol :(( i'll wait for comment

Reply:
See #149